demos/LibuvWithNonAllocatingFormatters/src/System/Net/Libuv/Handle.cs

```diff
@@ -50,7 +50,7 @@ public void Dispose()
             GC.SuppressFinalize(this);
         }
 
-        protected virtual void Dispose(bool disposing)
+        public virtual void Dispose(bool disposing)
```

Comment:
Ah, good catch. It's a left over from an experiment I did and then dropped. I will fix that with the next commit.